cert.go

```diff
@@ -217,7 +225,15 @@ func (opts *CertificateOptions) PutCertRequestFromMemory(wd depot.Depot) error {
 		return errors.Wrap(err, "problem getting formatted name")
 	}
 
-	if depot.CheckCertificateSigningRequest(wd, formattedName) || depot.CheckPrivateKey(wd, formattedName) {
+	csrExists, err := CheckCertificateSigningRequestWithError(wd, formattedName)
+	if err != nil {
+		return err
```

Comment:
Wrap error for context.

Reply:
I already wrap these errors in `pkix.go`.